code/modules/surgery/organs/lungs.dm

```diff
@@ -88,6 +88,15 @@
 
 	var/crit_stabilizing_reagent = /datum/reagent/medicine/epinephrine
 
+///Simply exists so that you don't keep any alerts from your previous lack of lungs.
+/obj/item/organ/internal/lungs/Insert(mob/living/carbon/receiver, special = FALSE, drop_if_replaced = TRUE)
+	receiver.clear_alert(ALERT_NOT_ENOUGH_OXYGEN)
+	receiver.clear_alert(ALERT_NOT_ENOUGH_CO2)
```

Comment:
This gives me pause. Is there a case where you would have these alerts beforehand, get new lungs, and they would still be relevant? Are they going to clear, delay, then add themselves back?

Reply:
Hmm? They would add themselves back almost immediately, pretty sure. I might have to test that though.

Reply:
I tested it, it gave it back to me less than a second after.

Reply:
Alright. I was concerned about the UX but if it's less than a second then I suppose that's fine